### ISSUE TITLE
[radio-group] mouse click fail to activate

### ISSUE THREAD

**Issue by yinonov** [Vonage/vivid-3#803]
mouse click on radio button within a radio-group does not activate the radio group. It does work with the keyboard when tabbed onto.

the expectation is that the mouse click will set focus on the group to allow the radio button to be navigated with the keyboard.

this could be caused by a css property. listbox had a similar issue and was fixed by setting a css property. ask @rinaoka @rachelbt for details.

**Comment by rinaok** [comment]
@olaf-k The problem with the listbox was solved by adding:

`:host {
	display: block;
}`

**Comment by yinonov** [comment]
> @olaf-k The problem with the listbox was solved by adding:
> 
> 
> 
> `:host {
> 
> 	display: block;
> 
> }`

This can always be abused by consumers so if finding an alternative it's better. Having css affecting functionality can lead to many issues. Try setting display flow interiorly. Might it also be related to focus delegation?

**Comment by olaf-k** [comment]
> (...) Might it also be related to focus delegation?

yup, in the radio actually. the fix is done, I'm adding a test atm.
